app/src/main/java/com/dsu/poc/repository/login/ValidationRepository.kt

```diff
@@ -0,0 +1,54 @@
+package com.dsu.poc.repository.login
+
+import android.app.Application
+import android.text.TextUtils
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+
+class ValidationRepository(application: Application) {
```

Comment:
why is validation a part of the repository? validation is business logic. it should be a part of view model or utils function

Reply:
done